lib/db.ts

```diff
@@ -1,9 +1,4 @@
-import { MongoClient } from "mongodb";
-import { mongoDBURI } from "/app/config";
+import { PrismaClient } from "@prisma/client";
 
-const options = {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-};
-
-export default new MongoClient(mongoDBURI, options);
+let dbClient = new PrismaClient()
```

Comment:
it is always seductive to use `let`, but unless it's _strictly_ necessary, I would always default to `const`.